src/style.scss

```diff
@@ -0,0 +1,148 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  padding: 0 16px;
+  width: 200px;
+  height: 406px;
+  border: 1px solid rgba(243, 243, 243, 1);
+
+  &__icon {
+    display: flex;
+    justify-content: center;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    img {
```

Comment:
not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/582c99f8-c7d4-4b32-ae97-6e972d60eafc)

you don't need `card__icon` div wrapper at all